hack/build/bazel-docker.sh

```diff
@@ -93,16 +109,24 @@ _rsync \
     ${CDI_DIR}/ \
     "rsync://root@127.0.0.1:${RSYNCD_PORT}/build"
 
-volumes="-v ${BUILDER_VOLUME}:/root:rw,z"
 
-# append .docker directory as volume
-mkdir -p "${HOME}/.docker"
-volumes="$volumes -v ${HOME}/.docker:/root/.docker:ro,z"
+if [ "${KUBEVIRTCI_RUNTIME}" != "podman" ]; then
+    volumes="-v ${BUILDER_VOLUME}:/root:rw,z"
+    # append .docker directory as volume
+    mkdir -p "${HOME}/.docker"
+    volumes="$volumes -v ${HOME}/.docker:/root/.docker:ro,z"
+else
+    volumes="-v ${BUILDER_VOLUME}:/root:rw,z,exec"
+fi
 
 # Ensure that a bazel server is running
 
 if [ -z "$(docker ps --format '{{.Names}}' | grep ${BAZEL_BUILDER_SERVER})" ]; then
-    docker run --network host -d ${volumes} --security-opt label:disable --name ${BAZEL_BUILDER_SERVER} -w "/root/go/src/kubevirt.io/containerized-data-importer" --rm ${BUILDER_IMAGE} hack/build/bazel-server.sh
+    if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
+        docker run --network host -d ${volumes} --security-opt label=disable --name ${BAZEL_BUILDER_SERVER} -w "/root/go/src/kubevirt.io/containerized-data-importer" --rm ${BUILDER_IMAGE} hack/build/bazel-server.sh
+    else
+        docker run --network host -d ${volumes} --security-opt label:disable --name ${BAZEL_BUILDER_SERVER} -w "/root/go/src/kubevirt.io/containerized-data-importer" --rm ${BUILDER_IMAGE} hack/build/bazel-server.sh
+    fi
```

Comment:
Are these two commands supposed to be different?

Reply:
They are different, look at the --security-opt label(:/=)disable